csharp-tree-sitter.el

```diff
@@ -261,19 +263,20 @@
 (defgroup csharp-mode-indent nil "Indent lines using Tree-sitter as backend"
   :group 'tree-sitter)
 
-(defcustom csharp-mode-indent-offset 4
-  "Indent offset for csharp-mode"
+(defcustom csharp-indent-offset 4
```

Comment:
Rename this to `csharp-tree-sitter-mode-indent-offset` or `csharp-tree-sitter-indent-offset`. Also, please change the docstring to something more correct :)